config/fund_loader_config/cyp/cyp_r1.py

```diff
@@ -0,0 +1,204 @@
+        "form_name_json": {"en": "proposed-services-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.4",
+    },
+    {
+        "section_name": {"en": "3.5 Working in partnership", "cy": ""},
+        "form_name_json": {"en": "working-in-partnership-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.5",
+    },
+    {
+        "section_name": {"en": "3.6 Proposal sustainability", "cy": ""},
+        "form_name_json": {"en": "proposal-sustainability-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.4.6",
+    },
+    {
+        "section_name": {"en": "4. Outputs and outcomes", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5",
+        "weighting": 15,
+    },
+    {
+        "section_name": {"en": "4.1 Outputs and outcomes", "cy": ""},
+        "form_name_json": {"en": "outputs-and-outcomes-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.5.1",
+    },
+    {
+        "section_name": {"en": "5. Risk and deliverability", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6",
+        "weighting": 15,
+    },
+    {
+        "section_name": {"en": "5.1 Risk and deliverability", "cy": ""},
+        "form_name_json": {"en": "risk-and-deliverability-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.6.1",
+    },
+    {
+        "section_name": {"en": "6. Value for money", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7",
+        "weighting": 15,
+    },
+    {
+        "section_name": {"en": "6.1 Funding required", "cy": ""},
+        "form_name_json": {"en": "funding-required-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7.1",
+    },
+    {
+        "section_name": {"en": "6.2 Building works", "cy": ""},
+        "form_name_json": {"en": "building-works-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7.2",
+    },
+    {
+        "section_name": {"en": "6.3 Match funding", "cy": ""},
+        "form_name_json": {"en": "match-funding-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.7.3",
+    },
+    {
+        "section_name": {"en": "7. Declarations", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.8",
+    },
+    {
+        "section_name": {"en": "7.1 Declarations", "cy": ""},
+        "form_name_json": {"en": "declarations-ns", "cy": ""},
+        "tree_path": f"{APPLICATION_BASE_PATH}.8.1",
+    },
+]
+
+fund_config = {
+    "id": CYP_FUND_ID,
+    "name_json": {
+        "en": "Children and Young People Resettlement Fund",
+        "cy": "",
+    },
+    "title_json": {
+        "en": "funding to resettle children and young people in England",
+        "cy": "",
+    },
+    "short_name": "CYP",
+    "description_json": {"en": "", "cy": ""},
+    "welsh_available": False,
+    "owner_organisation_name": "Department for Levelling Up, Housing and Communities",
+    "owner_organisation_shortname": "DLUHC",
+    "owner_organisation_logo_uri": DLUHC_LOGO_PNG,
+}
+
+round_config = [
```

Comment:
Could we try and be explicit here - so include some of the new booleans from the round model even if they’re false? I.e the Boolean for showing the file upload section, etc?

Reply:
needs workflow change as discussed 👍🏾